homeassistant/components/xiaomi_aqara/binary_sensor.py

```diff
@@ -140,20 +140,9 @@ class XiaomiBinarySensor(XiaomiDevice, BinarySensorEntity):
     def __init__(self, device, name, xiaomi_hub, data_key, device_class, config_entry):
         """Initialize the XiaomiSmokeSensor."""
         self._data_key = data_key
-        self._device_class = device_class
-        self._density = 0
```

Comment:
Note: I removed `self._density` because it is initialised with different values in all child classes - so there is no point initilialising it in the base class and it actually causes mypy issues